README.md

```diff
@@ -460,7 +460,7 @@ Table of Contents
   * [mailcatcher.me](https://mailcatcher.me/) — Catches mail and serves it through a web interface
   * [mailchimp.com](https://mailchimp.com/) — 2,000 subscribers and 12,000 emails/month free
   * [MailerLite.com](https://www.mailerlite.com) — 1,000 subscribers/month, unlimited email free
-  * [mailgun.com](https://www.mailgun.com/) — First 10,000 emails and 100 e-mail address validation is free each month
+  * [mailgun.com](https://www.mailgun.com/) — 5000 free emails/month for the first 3 months, maximum of 300 emails/day
```

Comment:
The main kicker of this is `You can only send to Authorized Recipients; and there is a maximum of 5 Authorized Recipients` which means I think we can completely remove them from the list

Reply:
> I think we can completely remove them from the list

Agreed, at this point their free tier is just a sandbox for those who want to eventually move to their paid tiers.